package.json

```diff
@@ -49,23 +49,24 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@capacitor/android": "^3.0.0",
-    "@capacitor/core": "^3.0.0",
-    "@capacitor/docgen": "^0.0.18",
-    "@capacitor/ios": "^3.0.0",
+    "@capacitor/android": "^4.0.0",
+    "@capacitor/cli": "^4.3.0",
+    "@capacitor/core": "^4.0.0",
+    "@capacitor/docgen": "^0.2.0",
+    "@capacitor/ios": "^4.0.0",
```

Comment:
the changes here to `@capacitor/core`, `@capacitor/android` and `@capacitor/ios` are really the key change - most of the rest of this PR is just example app changes to match.